src/room/InCallView.tsx

```diff
@@ -154,6 +174,46 @@ export function InCallView({
 
   const { hideScreensharing } = useUrlParams();
 
+  const makeConnectionStatesMap = useCallback(() => {
+    const newConnStates = new Map<string, ConnectionState>();
+    for (const participant of participants) {
+      const userCall = groupCall.getCallByUserId(participant.userId);
+      const feed = userMediaFeeds.find((f) => f.userId === participant.userId);
+      let connectionState = ConnectionState.ESTABLISHING_CALL;
+      if (feed && feed.isLocal()) {
```

Comment:
How does the fact we have a local feed mean we are connected?

Reply:
Well, it's our own feed, so I think that effectively means it's always connected? We could also say connection state is N/A for a local feed, although it would make it slightly more complex.

Reply:
Ah, sorry, I was thinking about this the wrong way